### PR TITLE
Add privacy policy link

### PR DESCRIPTION
**Pull request by DavidBiddle**
#### What problem does the pull request solve?
Reads a form's privacy policy URL and displays it in the footer if it's present.

Trello card: https://trello.com/c/aZtUyjei/16-add-link-to-privacy-information-to-forms-runner-in-footer-of-every-form-pageneeds-dev-review

#### Checklist

- [x] I've used the pull request template
- [x] I've linked this PR to the relevant issue (if mission work)
- [x] I've written unit tests for these changes (if code change)
- [n/a] I've updated the documentation in (If any documentation requires updating)
    - [n/a] README.md
    - [n/a] Elsewhere (please link)

### CODE REVIEW

File: app/controllers/form_controller.rb [hunk 1]
```diff
@@ -9,5 +10,12 @@ def show
 
   def submitted
     @form = Form.find(params.require(:form_id))
+    set_privacy_policy_url
+  end
+
+private
+
+  def set_privacy_policy_url
+    @privacy_policy_url = @form.privacy_policy_url
```
Comment (aliuk2012) [inline app/controllers/form_controller.rb:19]: 🤔  Is it worth maybe setting `@privacy_policy_url` from the main ApplicationController so we shouldn't have to do it in specific controllers. We could also set a default to our own privacy policy for instances like 404/500 pages

**Review by thomasiles — APPROVED** [review]
looks good!

File: app/controllers/form_controller.rb (hunk 1 above)
Comment (DavidBiddle) [reply app/controllers/form_controller.rb:19]: Probably a good idea but I'm running into something tricky locally with the `current_context` stuff, so I'm going to call it a day for now - can have another look when I'm back next week, or happy for someone else to pick this up in the meantime if we want to get it done by end of sprint.

Comment (aliuk2012) [reply app/controllers/form_controller.rb:19]: Ah I see what you mean, there is a lot of code hanging over from initial prototyping meaning we have things like `form_controller.rb` and `forms/form_controller.rb` and a few other controllers that we dont need any more.

I'm happy to push this through and we can tweak this as part of tidying up all the controllers just generally.

**Pull request merged**